src/server.js

```diff
@@ -44,6 +44,26 @@ const catchError = (fn) => async (req, res) => {
 
 }
 
+const ACCESS_CONTROL_ALLOW_ORIGIN = process.env.ACCESS_CONTROL_ALLOW_ORIGIN
+
+/**
+ * Attaches CORS headers to all requests.
+ * This is needed if the client is sending requests to Ackee
+ * from a different domain.
+ * @param {*} fn Micro request callback
+ * @returns {Function} Function which is called on each request made to the server
+ */
+const attachCORSHeaders = (fn) => async (req, res) => {
+	if (!ACCESS_CONTROL_ALLOW_ORIGIN) {
+		return await fn(req, res)
+	}
+	res.setHeader('Access-Control-Allow-Origin', ACCESS_CONTROL_ALLOW_ORIGIN)
+	res.setHeader('Access-Control-Allow-Credentials', true)
+	res.setHeader('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE,OPTIONS')
```

Comment:
This adds more methods and headers than required. Let's keep the amount low: https://github.com/electerious/Ackee/blob/master/docs/CORS%20headers.md